tools/mFold.md

```diff
@@ -0,0 +1,131 @@
+# mFold
+
+written by: Kevin Yuan
+
+[{estimated time} minute] This tool is for tutorial 11. mFold is one of the earliest tools for computational molecular biology, operating since 1995. It is primarily used for prediction of nucleic acid folding, which can be valuable in evaluating and comparing the structure of viruses.
```

Comment:
What is the estimated time?

No need to say it's for Tutorial 11, you're writing a tutorial here, not writing an assignment answer.